### PR TITLE
Fix alert message being dismissed automatically

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Token/Controllers/TabBarController.swift [hunk 1]
```diff
@@ -106,7 +106,7 @@ open class TabBarController: UITabBarController {
     private func presentAddressChangeAlertIfNeeded() {
         guard UserDefaults.standard.bool(forKey: AddressChangeAlertShown) == false else { return }
 
-        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```
Comment (Elland) [inline Token/Controllers/TabBarController.swift]: I think we should remove this delay and remove the VC being dismissed inside `didCreateUser`.

Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: you think it is better to show alert without delay at all ?

Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: i think it feel better with delay 😄

Comment (Elland) [inline Token/Controllers/TabBarController.swift]: I am assuming this only shows after you've signed in/signed up, right?

Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: i dnt see point of removing dismissal there, as i described, with new flow we will need to dismiss sign in scene :/

Comment (Elland) [inline Token/Controllers/TabBarController.swift]: But that is not dismissing sign in, that is dismissing anything that is being currently presented. 😜

Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: in this case i need to make this change in sign in flow pr . are you ok with that ?

Comment (Elland) [inline Token/Controllers/TabBarController.swift]: Show me what you have in mind.

Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: https://github.com/tokenbrowser/token-ios-client/pull/265/files#diff-73a30b83c4db1a4cc6e681e79bb192a5R137

here i will dismiss and only then will remove this line

Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: regarding showing only after  signed in/signed up - nope, oppositely, it shows if you are using app already

Comment (Elland) [inline Token/Controllers/TabBarController.swift]: 👌🏽

**Comment by Elland** [comment]
@yuliaveres ready for merging?

**Comment by yuliaveres** [comment]
@Elland  Yes !

**Pull request merged**